src/main/java/br/eti/kinoshita/testlinkjavaapi/TestCaseService.java

```diff
@@ -879,4 +873,28 @@ protected void assignTestCaseExecutionTask(Integer testPlanId, String testCaseEx
         }
     }
 
+    /**

```

Comment:
I reckon this is the only change. I've been postponing merging this one simply because it appeared to have too many changes. But apparently the other ones are only formatting, and not necessary for this new feature.

It is always easier to review pull requests that include the minimum necessary. Will edit the commit manuallya to remove the other changes.

Thanks, releasing soon, and sorry for the delay.

Bruno